contracts/freeportParts/Issuance.sol

```diff
@@ -84,6 +88,13 @@ abstract contract Issuance is ERC20Adapter {
         return id;
     }
 
+    function metadataURL(uint nftId, string calldata postfix) public view returns (string memory) {
```

Comment:
What is this function? Why not use the standard that is already there?

[function uri here](https://github.com/Cerebellum-Network/Freeport-Smart-Contracts/blob/d3be509cb3b41827d6f2fb6e81be6224b15f8835/contracts/token/ERC1155/ERC1155Upgradeable.sol#L67).

All that is needed is to change the value of the _uri variable.

Also, with this standard, there is no need to render the URL in the contract code, it can be done by the client (replacing `{id}` in the template).

See the standard: https://eips.ethereum.org/EIPS/eip-1155#metadata